RF24_config.h

```diff
@@ -52,20 +52,20 @@
     // The includes.h file defines either RF24_RPi, MRAA, LITTLEWIRE or RF24_SPIDEV and includes the correct RF24_arch_config.h file
     #include "utility/includes.h"
 
-//ATTiny    
-#elif defined (__AVR_ATtiny25__) || defined (__AVR_ATtiny45__) || defined (__AVR_ATtiny85__) || defined (__AVR_ATtiny24__) || defined (__AVR_ATtiny44__) || defined (__AVR_ATtiny84__) || defined (__AVR_ATtiny2313__) || defined (__AVR_ATtiny4313__) || defined (__AVR_ATtiny861__)    
+//ATTiny
+#elif defined (__AVR_ATtiny25__) || defined (__AVR_ATtiny45__) || defined (__AVR_ATtiny85__) || defined (__AVR_ATtiny24__) || defined (__AVR_ATtiny44__) || defined (__AVR_ATtiny84__) || defined (__AVR_ATtiny2313__) || defined (__AVR_ATtiny4313__) || defined (__AVR_ATtiny861__) || defined (__AVR_ATtinyX5__) || defined (__AVR_ATtinyX4__) || defined (__AVR_ATtinyX313__) || defined (__AVR_ATtinyX61__)
```

Comment:
for compatibility with other ATTinyCores (that have a SPI library), I've simply appended this line to use macros defined in the SpenceKonde ATTinyCore (`__AVR_ATTinyX5__` is declared instead of `__AVR_ATTiny85__` or `__AVR_ATTiny45__`).

Since the RF24 library depends on SpencKonde's SPI library, should I have simply replaced this line instead of appending it?